src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostModal/ActualCostModal.test.js

```diff
@@ -0,0 +1,193 @@
+    });
+
+    it('should render fee fine owner type', () => {
+      expect(screen.getByText(initialProps.actualCost.actualCostRecord.feeFine.type)).toBeVisible();
+    });
+
+    it('should render "actualCostToBill" field', () => {
+      expect(screen.getByText(messageIds.actualCostToBill)).toBeVisible();
+    });
+
+    it('should render "actualCostToBill" field with correct props', () => {
+      const expectedProps = {
+        required: true,
+        value: initialProps.actualCost.additionalInfo.actualCostToBill,
+      };
+
+      expect(TextField).toHaveBeenCalledWith(expect.objectContaining(expectedProps), {});
+    });
+
+    it('should render "additionalInformationForStaff" textarea', () => {
+      expect(screen.getByText(messageIds.additionalInformationForStaff)).toBeVisible();
+    });
+
+    it('should render "additionalInformationForStaff" textarea with correct props', () => {
+      const expectedProps = {
+        value: initialProps.actualCost.additionalInfo.additionalInformationForStaff,
+      };
+
+      expect(TextArea).toHaveBeenCalledWith(expect.objectContaining(expectedProps), {});
+    });
+
+    it('should render "additionalInformationForPatron" textarea', () => {
+      expect(screen.getByText(messageIds.additionalInformationForPatron)).toBeVisible();
+    });
+
+    it('should render "additionalInformationForPatron" textarea with correct props', () => {
+      const expectedProps = {
+        value: initialProps.actualCost.additionalInfo.additionalInformationForPatron,
+      };
+
+      expect(TextArea).toHaveBeenCalledWith(expect.objectContaining(expectedProps), {});
+    });
+
+    describe('footer', () => {
+      it('should render "Continue" button label', () => {
+        expect(screen.getByText(messageIds.continueButton)).toBeVisible();
+      });
+
+      it('should trigger "setActualCostModal" and "setActualCostConfirmModal" with correct data', () => {
+        const continueButton = screen.getByTestId(testIds.continueActualCost);
+
+        fireEvent.click(continueButton);
+
+        expect(initialProps.setActualCostModal).toHaveBeenCalledWith(ACTUAL_COST_MODAL_DEFAULT);
+        expect(initialProps.setActualCostConfirmModal).toHaveBeenCalledWith({
+          isOpen: true,
+        });
+      });
+
+      it('should render "Cancel" button label', () => {
+        expect(screen.getByText(messageIds.cancelButton)).toBeVisible();
+      });
+
+      it('should trigger "setActualCostModal" and "setActualCost" with correct data', () => {
+        const cancelButton = screen.getByTestId(testIds.cancelActualCost);
+
+        fireEvent.click(cancelButton);
+
+        expect(initialProps.setActualCostModal).toHaveBeenCalledWith(ACTUAL_COST_MODAL_DEFAULT);
+        expect(initialProps.setActualCost).toHaveBeenCalledWith(ACTUAL_COST_DEFAULT);
+      });
+    });
+  });
+
+  describe('when actual cost type equals "doNotBill"', () => {
```

Comment:
Could you please check can we check that additionalInformationForPatron are absent?

Reply:
Added initial test